ee/tables/jwt/table.go

```diff
@@ -0,0 +1,176 @@
+package kolide_jwt
+
+import (
+	"context"
+	"crypto"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/kolide/launcher/ee/dataflatten"
+	"github.com/kolide/launcher/ee/tables/dataflattentable"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+var (
+	ErrMissingKeyId     = errors.New("no key id found in the JWT header")
+	ErrMatchingKeyId    = errors.New("no key id matched the JWT header key id")
+	ErrParsingPemBlock  = errors.New("error parsing PEM block containing the public key")
+	ErrParsingPublicKey = errors.New("error parsing the public key from the PEM block")
+)
+
+type Table struct {
+	slogger *slog.Logger
+}
+
+func TablePlugin(slogger *slog.Logger) *table.Plugin {
+	columns := dataflattentable.Columns(
+		table.TextColumn("path"),
+		table.TextColumn("signature_key"),
```

Comment:
I want to quibble over the name, but I'm not sure what. Maybe `validation_keys` or `signing_keys` or ?

Reply:
Yeah `signature_key` was more or less a placeholder, so I'm good to change it but I'm not sure what would feel best here. I personally like `signing_keys`, so maybe that?

Reply:
I've changed it to `signing_keys`, but I'm still open to other naming